rootfs-builder/debian/rootfs_lib.sh

```diff
@@ -0,0 +1,27 @@
+# - Arguments
+#
+# Copyright (c) 2018 SUSE
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+#
+# rootfs_dir=$1
```

Comment:
Why is this line here?

Reply:
Thanks for the review. Most of the extra comment lines comes from a copy/paste of the ubuntu folder. And I agree, they are not needed at all.

Reply:
Can you create a separate PR to remove them too? They must have got in by mistake.

Reply:
Hi @ydjainopensource I opened a new issue for this: #171

I'll follow up with a new PR, once the current PRs pipeline starts to flush :)

Reply:
:laughing: - So many jokes!

I think @grahamwhaley is sitting on the Kata... cough... "Throne" this week so lets see if he can unblock it without having to take a plunger to it... :smile: